beetmoverscript/task.py

```diff
@@ -148,31 +132,12 @@ def get_release_props(context, platform_mapping=STAGE_PLATFORM_MAP):
     """determined via parsing the Nightly build job's balrog_props.json and
```

Comment:
We need to fix the docstring too, since it no longer reflects `balrog_props`. See my below comment

Reply:
good catch!

Reply:
We still need to amend the docstring here